Ports/qt6-qt5compat/package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='qt6-qt5compat'
+version='6.2.3'
+workdir="qt5compat-everywhere-src-${version}"
+useconfigure='true'
+files="https://download.qt.io/official_releases/qt/$(cut -d. -f1,2 <<< ${version})/${version}/submodules/qt5compat-everywhere-src-${version}.tar.xz qt6-qt5compat-${version}.tar.xz 1cf89198cf2cf8a5c15336ccd69fa1f39b779feb64117d6bbf5509c21c123f53"
+auth_type='sha256'
+depends=(
+    'qt6-qtbase'
+    'libiconv'
+)
+
+configure() {
+    export LDFLAGS='-liconv'
+    run cmake -GNinja \
+        -DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt \
+        -DCMAKE_CROSSCOMPILING=ON \
+        -DQT_BUILD_TOOLS_WHEN_CROSSCOMPILING=ON \
+	-DQT_HOST_PATH=$(qmake6 -query QT_HOST_PREFIX) \
```

Comment:
This line seems to be indented with one tab instead of eight spaces.

Reply:
Thanks!